packages/article/src/Infrastructure/Sulu/Content/ArticleDataProvider.php

```diff
@@ -97,10 +98,10 @@ public function resolveDataItems(array $filters, array $propertyParameter, array
         $result = [];
         foreach ($articles as $article) {
             $dimensionContent = $this->contentManager->resolve($article, $dimensionAttributes);
-            $result[] = [
+            $result[] = new ArrayAccessItem($article->getId(), [
                 'id' => $article->getId(),
                 'title' => $dimensionContent->getTitle(),
-            ];
+            ], $article);
```

Comment:
hopefully we will remove this classes soon but currently phpstan requires ResourceItemInterface